src/api/goods.ts

```
@@ -4,13 +4,34 @@ import { Good } from '../types/Good';
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
 export function getAll(): Promise<Good[]> {
-  return fetch(API_URL).then(response => response.json());
+  return fetch(API_URL)
+    .then(response => response.json())
+    .catch(() => {
+      return 'Couldnt load';
+    });
```

Instead of returning a string error message, consider throwing an error or returning a rejected promise. This will allow the calling component to handle errors more effectively and maintain type consistency.